### ISSUE TITLE
Add "attendees" table

### ISSUE THREAD

**Issue by jberger** [mojoconf/MCT#10]
The attendees table is used to see which users have attended which conferences and the attendance list of any given conference. It can also track per-conference user information like payment status.

Note that I had this table planned before eventbrite integration was assumed.

**Comment by jhthorsen** [comment]
Maybe #8 isn't future after all...

https://developer.eventbrite.com/docs/webhooks/
http://developer.eventbrite.com/docs/webhooks-summary/

I was thinking we could add a hook when an order is placed, and populate the attendees table automatically.

**Comment by jhthorsen** [comment]
I think we can do without this for now. We can rather import the users later on from eventbrite to add more data to MCT.

**Comment by jhthorsen** [comment]
Also: https://developer.eventbrite.com/docs/event-attendees/

It returns a structure something like this:
```json
{
  "pagination": { "object_count": 1, "page_number": 1, "page_size": 50, "page_count": 1 }, 
  "attendees": [
    {
      "team": null, 
      "resource_uri": "https://www.eventbriteapi.com/v3/events/15957581556/attendees/507205665/", 
      "id": "507205665", 
      "changed": "2015-02-28T15:26:19Z", 
      "created": "2015-02-28T15:26:12Z", 
      "quantity": 1, 
      "ticket_class_id": "34013864", 
      "profile": {
        "first_name": "Jan Henning", 
        "last_name": "Thorsen", 
        "email": "jhthorsen@cpan.org", 
        "name": "Jan Henning Thorsen", 
        "addresses": {}
      }, 
      "barcodes": [ ... ],
      "answers": [ ... ], 
      "costs": {
        "payment_fee": { "currency": "USD", "display": "$0.00", "value": 0 }, 
        "gross": { "currency": "USD", "display": "$0.00", "value": 0 }, 
        "eventbrite_fee": { "currency": "USD", "display": "$0.00", "value": 0 }, 
        "tax": { "currency": "USD", "display": "$0.00", "value": 0 } 
      }, 
      "checked_in": false, 
      "cancelled": false, 
      "refunded": false, 
      "affiliate": null, 
      "status": "Attending", 
      "event_id": "15957581556", 
      "order_id": "401444024", 
      "event": { ... }, 
      "order": { ... } 
    } 
  ]
}
```

**Comment by jhthorsen** [comment]
Closed by 8221a55a54d3b26296e7e1e6bc90b8dd30254f47.
